tabletop/core/index.js

```diff
@@ -15,7 +15,7 @@ var core = Object.assign({
     NextPlayerView: require('./next_player_view'),
     Player: require('./player'),
     StartView: require('./start_view'),
-    StyleSheets: require('../../src/checkers/style.css'),
+//    StyleSheets: require('../../src/checkers/style.css'),
```

Comment:
@andrewjmeier I still can't run webpack with this line not commented. I didn't want to uncomment and merge when i haven't tested it, so wasn't sure what to do with this line